CHANGELOG.md

```diff
@@ -2,510 +2,321 @@
 
-🆕 New features:
+- dark-blue
+- dark-grey
+- mid-grey
+- light-grey
 
-- IE 8 now falls back to Arial rather than Transport
+5 Sass colour variables no longer exist. Replace the following colour variables if you're using Sass:
 
-  All of the browsers we support use the WOFF or WOFF2 fonts, except for IE 8
-  which requires Embedded Open Type (.eot). The EOT version of the font is
-  considerably larger, which impacts performance for users who are likely to be
-  on older machines.
+Colour variable removed|Suggested replacement
+| --- | --- |
+bright-red|red
+grey-1|dark-grey
+grey-2|mid-grey
+grey-3|light-grey
+grey-4|light-grey
 
-  Given IE8 only represents a very small percentage of traffic we get, we now
-  fallback to Arial instead.
+You should check the [contrast ratio of your colours](https://design-system.service.gov.uk/styles/colour#colour-contrast).
 
-  Thanks to [@Nooshu](https://github.com/Nooshu) for their
-  [recommendation to drop EOT files for older IE](https://github.com/alphagov/govuk-frontend/pull/1356#issuecomment-494727525).
+If you're not using Sass, change the values of the following colours:
 
-  ([PR #1434](https://github.com/alphagov/govuk-frontend/pull/1434))
+Colour name|Old value|Replace with
+| --- | --- | --- |
+purple|#2e358b|#4c2c92
+red|#b10e1e|#d4351c
+yellow|#ffbf47|#ffdd00
+green|#006435|#00703c
+light-blue|#2b8cc4|#5694ca
+blue|#005ea5|#1d70b8
 
-- Checkboxes and radios use a new focus state with a thicker border. The
-  transparent outline, previously required to show the focus state when custom
-  colour schemes are used, has been removed.
+We've also changed the background of the following components:
 
-  ([PR #1316](https://github.com/alphagov/govuk-frontend/pull/1316))
+- buttons - `green` instead of a custom green
+- confirmation panels - `green` instead of `turquoise`
+- links in their hover state - `dark-blue` instead of `light-blue`
 
-- A new setting `$govuk-use-legacy-palette` has been added, which by default
-  will be true if any of the `$govuk-compatibility-*` settings are true.
+If you're using legacy projects like GOV.UK Elements, you can keep your current colours by [turning on compatibility mode](https://github.com/alphagov/govuk-frontend/docs/installation/compatibility.md).
 
-  When set to `true`, the existing colour palette from v2.x of GOV.UK Frontend
-  will be used.
+Read our [blog post about why we changed the colour palette](xxxx)
 
-  ([PR #1288](https://github.com/alphagov/govuk-frontend/pull/1288))
+[Pull request #1288: Update colour palette](https://github.com/alphagov/govuk-frontend/pull/1288).
 
-- The `govuk-colour` function has been updated to add a `$legacy` argument,
-  which allows you to specify a colour (either a literal, or a name of a colour
-  from the legacy palette) to use when `$govuk-use-legacy-palette` is true.
+### Check the new font
```

Comment:
Would it be worth saying that the font itself is unchanged, this is just a new version of it.

Reply:
Ah good point - would '### Check the new version of the font' work better?

Reply:
Yeah that makes sense 👍

Reply:
Updated in https://github.com/alphagov/govuk-frontend/pull/1492/commits/217e12a762789d8040c56dc7831cebc0c9282801